volue_insight_timeseries/util.py

```diff
@@ -26,29 +27,38 @@
 
 # Frequency mapping from TS to Pandas
 _TS_FREQ_TABLE = {
-    'Y': 'AS',
+    'Y': 'YS',
     'S': '2QS',
     'Q': 'QS',
     'M': 'MS',
     'W': 'W-MON',
-    'H12': '12H',
-    'H6': '6H',
-    'H3': '3H',
-    'MIN30': '30T',
-    'MIN15': '15T',
-    'MIN5': '5T',
-    'MIN': 'T',
+    'H12': '12h',
+    'H6': '6h',
+    'H3': '3h',
+    'MIN30': '30min',
+    'MIN15': '15min',
+    'MIN5': '5min',
+    'MIN': 'min',
```

Comment:
Wouldn't it be simpler to use upper case like the original version, so you can avoid calling `v.upper()` on line 61?

Reply:
> Wouldn't it be simpler to use upper case like the original version, so you can avoid calling `v.upper()` on line 61?

This is what we put to pandas, so I decided for correctness of what pandas expect, instead of relying of pandas internal mapping (who knows, if at some point they make a differentiation between upper/lower case or throw a warning about improper case). That's why I've made it explicit the way it is in pandas.